src/query/service/src/global_services.rs

```diff
@@ -107,13 +110,25 @@ impl GlobalServices {
         SessionManager::init(config)?;
         LockManager::init()?;
         AuthMgr::init(config)?;
-        UserApiProvider::init(
-            config.meta.to_meta_grpc_client_conf(),
-            config.query.idm.clone(),
-            &config.query.tenant_id,
-            config.query.tenant_quota.clone(),
-        )
-        .await?;
+
+        // Init user manager.
+        // Builtin users and udfs are created here.
+        {
+            let built_in_users = BuiltinUsers::create(config.query.builtin.users.clone());
+            let built_in_udfs = BuiltinUDFs::create(config.query.builtin.udfs.clone());
+            let builtin = BuiltIn {
+                users: built_in_users.to_meta_auth_infos()?,
+                udfs: built_in_udfs.to_meta_udfs().await?,
```

Comment:
Provide additional context for this error, so the user can avoid encountering a parse error without knowing its origin.

Reply:
There is no need to raise errors when the global init, if the udf&user config is invalid, we should skip them and leave a error.